src/include/sof/audio/audio_stream.h

```diff
@@ -1040,6 +1040,14 @@ static inline void audio_stream_fmt_conversion(enum ipc4_bit_depth depth,
 	*frame_fmt = (depth >> 3) - 2;
 	*valid_fmt = (valid >> 3) - 2;
 
+	if (depth == 8) {
```

Comment:
"if (depth == 8 && valid == 8) {"

Wouldn't this be more accurate? It seems any other case would be an invalid configuration.

Reply:
I separated it to support also a 8 bit samples in 16 and 32 bit containers